shared/components/page-header/__stories__/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import PageHeader from '../PageHeader';
+
+const count = [].length;
```

Comment:
This doesn't really do anything, it just resolves to `0`.

Reply:
What does the story have to display? The exact amount of items in an array of time entries/clients/ team members? Or do i just use mock data?

Reply:
Considering it's a pretty basic component I would use a static title.

Reply:
```
const arrayOfItems = ['Item 1', 'Item 2', 'Item 3'];
const count = arrayOfItems.length;
```

Is this a suitable fix?

Reply:
That's not what I meant, I would just do `subtitle="2 entries"`. There's no use for defining an array and then counting the length. It's the same as writing `count = 3` in this case.

Reply:
Ok, done!